junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestExtensionTests.java

```diff
@@ -69,8 +69,8 @@ void streamsReturnedByProvidersAreClosedWhenCallingProvide() {
 		Stream<TestTemplateInvocationContext> stream = this.parameterizedTestExtension.provideTestTemplateInvocationContexts(
 			extensionContext);
 
-		//cause the stream to be evaluated
-		stream.count();
+		// cause the stream to be evaluated
+		long unused = stream.count();
```

Comment:
Hmmmm.... aren't we now going to see warnings about the `unused` variable being _unused_?